### PR TITLE
Rerender v3.3, update template

### PR DESCRIPTION
**Pull request by pmlandwehr**
Fixed pull req to rerender `3.3.0.23` using `py36`

### CODE REVIEW

**Comment by kwilcox** [comment]
You reset the build number to 0

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,19 @@
+{% set name = "billiard" %}
 {% set version = "3.3.0.23" %}
+{% set sha256 = "692a2a5a55ee39a42bcb7557930e2541da85df9ea81c6e24827f63b80cd39d0b" %}
+{% set build = 0 %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should be 3.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,15 +27,14 @@ test:
   imports:
     - billiard
     - billiard.dummy
-    - billiard.tests
-    - funtests
```
Comment (jakirkham) [inline recipe/meta.yaml]: Not sure that these should have been removed.

Comment (jakirkham) [inline recipe/meta.yaml]: Fixing in PR ( https://github.com/conda-forge/billiard-feedstock/pull/12 ).

**Pull request merged**